seqr/views/apis/saved_variant_api_tests.py

```diff
@@ -75,6 +75,7 @@
 
 class SavedVariantAPITest(AuthenticationTestCase):
     fixtures = ['users', '1kg_project']
+    databases = {'reference_data', 'default'}
```

Comment:
It is required by the new version of the Django to access the database in a different 'application'.